src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs

```diff
@@ -455,7 +455,7 @@ public bool ForEach(KeyValuePair<string, object> activityTag)
                     case string[] stringArray:
                         foreach (var item in stringArray)
                         {
-                            PooledList<OtlpCommon.KeyValue>.Add(ref this.Tags, CreateOtlpKeyValue(key, new OtlpCommon.AnyValue { StringValue = item }));
+                            PooledList<OtlpCommon.KeyValue>.Add(ref this.Tags, CreateOtlpKeyValue(key, new OtlpCommon.AnyValue { StringValue = item ?? string.Empty }));
```

Comment:
I think for OTLP it would be better to use the native OTLP null value.

Reply:
See https://github.com/open-telemetry/opentelemetry-proto/blob/v0.7.0/opentelemetry/proto/common/v1/common.proto#L28-L29

Reply:
This is a great observation, I had previously thought that CreateOtlpKeyValue() didn't ccept nulls for values. Will adapt PR

Reply:
@Oberon00 I merged this PR, but could you confirm if the recent commits addressed your comment?